plugins/coral-plugin-viewing-options/client/containers/ViewingOptions.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import ViewingOptions from '../components/ViewingOptions';
+import {openViewingOptions, closeViewingOptions} from 'coral-embed-stream/src/actions/stream';
```

Comment:
We need to provide this actions through the `coral-plugin-api` in a folder called `alpha` or `beta` if we are more confident that it doesn't change.